### ISSUE TITLE
Display footer widgets without requiring widgets in all four areas

### ISSUE THREAD

**Issue by fjarrett** [LiftUX/ascension-theme#21]
I noticed that widgets do not appear in the Footer unless all four sidebars contain widgets, this is potentially confusing behavior and something users may not expect.

I think the footer area column widths need to dynamically change (25%, 33%, 50%, 100%) based on which sidebars contain widgets.

**Comment by schrapel** [comment]
My guess is this was done to not output excess blank space on the `.footer-widget-area` when no sidebars are active. I have changed it to check for any active sidebars or social menu. I only show the `.site-title` and `.site-description` if footer-4 or social menu is active. Is that ok? I think it looks kinda bare with a column with only the site title / description.

I counted the active sidebars and added a class to the widget area. I could have done it without any php and flex box but I'm not confident about complete browser support (it has lots of issues with IE11). The css that could be used instead is below.

```
display: flex;
justify-content: space-between;
```
